.github/workflows/build-test-release.yml

```diff
@@ -1,126 +0,0 @@
-name: Build, Test, and Release Shared GitHub Actions
-
-on:
-  pull_request:
-  push:
-    branches: ["main"]
-
-permissions:
-  contents: write
-
-env:
-  CARGO_TERM_COLOR: always
-
-jobs:
-  lint:
-    name: Lint
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Update Rust toolchain
-        run: rustup update
-      - name: Rust Cache
-        uses: Swatinem/rust-cache@v2.5.1
-      - name: Clippy
-        run: cargo clippy --all-targets --all-features -- --deny warnings
-      - name: rustfmt
-        run: cargo fmt -- --check
-      - name: Check docs
-        run: RUSTDOCFLAGS="-D warnings" cargo doc --all-features --document-private-items --no-deps
-
-  test:
-    name: Test
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Update Rust toolchain
-        run: rustup update
-      - name: Rust Cache
-        uses: Swatinem/rust-cache@v2.5.1
-      - name: Test
-        run: cargo test --all-features
-
-  check-bootstrap:
-    name: Check Bootstrap Script
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Setup Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: lts/*
-          cache: npm
-      - name: Install Node dependencies
-        run: npm ci
-      - name: Rebuild bootstrap script
-        run: npm run build
-      - name: Check bootstrap script for uncommitted changes
-        run: if [ -n "$(git status --porcelain)" ]; then exit 1; fi
-
-  release:
```

Comment:
With the removal of the old release workflow (plus once #97 is merged), I believe all of the following can be deleted:

```
.github/actions/<everything except the new install cli action>
.github/bootstrap/*
package-lock.json
package.json
tsconfig.json
```

Plus the `node_modules` entry in `.gitignore`.

Reply:
done + updated the README to reflect the new setup